pkg/plugin/plugin.go

```diff
@@ -275,14 +276,32 @@ func (w *flushWatcher) refreshPluginState() error {
 	}
 	return nil
 }
-
 func (w *flushWatcher) watchLoop() {
-	watchChan := w.etcd.Watch(w.ctx, w.path)
+	const reWatchInterval = time.Second * 5
```

Comment:
FYI, it seems that this interval hasn't been used. Is it expected? Should it be implemented as a timer ?

Reply:
The lost sleep has been added in PR: https://github.com/pingcap/tidb/pull/49292 . Thanks a lot